App.js

```diff
@@ -16,17 +16,26 @@ function SideBar() {
   return (
     <div className="NavBarCol">
       <col />
-      <button className="Icon"/>
-      <button className="Icon"/>
-      <button className="Icon"/>
-      <button className="Icon"/>
-      <button className="Icon"/>
-      <button className="Icon"/>
-      <button className="Icon"/>
-      <button className="Icon"/>
-      <button className="Icon"/>
-      <button className="Icon"/>
-
+      <div classNAme="ButtonRow">
```

Comment:
Typo in `classNAme`

Reply:
When it comes to naming CSS classes, there are a couple of factors.

First, if you are using just a single CSS file to style everything, then you need a way to make sure that you names don't conflict as the application grows.

For example, you might have page 1 that has a "ButtonRow" and a page 2 that also has a "ButtonRow," but you want them to be styled differently on each page. You would have to update the name of one of the classes so that they don't conflict (i.e. "ButtonRow1" and "ButtonRow2"). You can see how this can get out of hand quickly. So, people have introduced _naming conventions_ to help. For example, there is the [BEM](http://getbem.com/naming/) (block, element, modifier) naming convention. The downside to a naming convention is that it is a lot to memorize.

However, there is an ever better solution, CSS Modules. CSS Modules are CSS classes that can be imported into a JavaScript file. As they are imported, the tooling automatically appends a unique postfix to it (i.e. someClass becomes --> someClass-4u3984538. This allows you to reuse the same class names across files without worrying about a conflict. Here's how to use it with Create React App (the tooling behind a React app): https://dev.to/0x96f/using-css-modules-with-create-react-app-22hd